src/pyodide/BUILD.bazel

```diff
@@ -8,19 +8,19 @@ python_bundles()
 
 alias(
     name = "pyodide.capnp.bin",
-    actual = "pyodide.capnp.bin@rule@0.26.0a2",
```

Comment:
@hoodmane could use even more bazel magic here 😄

Reply:
Added in https://github.com/cloudflare/workerd/pull/3991